server/config/toml.go

```diff
@@ -86,52 +86,6 @@ iavl-disable-fastnode = {{ .BaseConfig.IAVLDisableFastNode }}
 # The fallback is the db_backend value set in CometBFT's config.toml.
 app-db-backend = "{{ .BaseConfig.AppDBBackend }}"
 
-###############################################################################
```

Comment:
wouldn't this actually break existing telemetry users? like without this its not exactly deprecated but fully broken?

Reply:
did more testing, if we started a new app with this removed, then we'd lose access to all cosmos SDK metrics currently wired with the legacy go-metrics system.